server/utils/telemetry/rudder.go

```diff
@@ -0,0 +1,47 @@
+package telemetry
+
+import rudder "github.com/rudderlabs/analytics-go"
+
+const (
+	rudderDataPlaneURL = "https://pdat.matterlytics.com"
+	rudderWriteKey     = "1d5bMvdrfWClLxgK1FvV3s4U1tg"
```

Comment:
@larkox We need to change this, right?

Reply:
If we want to use the library in experimental, you can follow directly the instructions [there](https://github.com/mattermost/mattermost-plugin-api/blob/master/experimental/telemetry/doc.go). If we were to keep this implementation, the only difference with the documentaion would be to set the right path to the `LDFLAGS` part.

The makefile may not use LDFLAGS. In that case, you can take a look on how it is done on the PRs for [Zoom](https://github.com/mattermost/mattermost-plugin-zoom/pull/148) or [Todo](https://github.com/mattermost/mattermost-plugin-todo/pull/101).